### PR TITLE
[FAB-17408] Move first app tutorial (fabcar) to test network

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: NIKHIL E GUPTA <negupta@us.ibm.com>

#### Type of change

- Documentation update

#### Description

Move first app tutorial to test network from first network

#### Related issues

The tutorial update corresponds with this update to the fabcar sample: https://github.com/hyperledger/fabric-samples/pull/103/files
The two should be merged together.

This PR also removes the understanding fabcar doc (not in the table of contents) to address this issue: https://jira.hyperledger.org/browse/FAB-15354. Most information in the removed doc was in other places, but one helpful sentence was added to the writing first app doc.

### CODE REVIEW

File: docs/source/write_first_app.rst [hunk 1]
```diff
@@ -316,9 +289,15 @@ ledger.
...
+new terminal and navigate to the JavaScript version of the FabCar Smart contract
+inside the ``fabric-samples`` repository:
+
+.. code:: bash
+
+  cd fabric-samples/chaincode/fabcar/javascript/lib
+
+Open the ``fabcar.js`` file in a text editor editor.
 
 See how our smart contract is defined using the ``Contract`` class:
```
[the remark this answers is not in the document]
Comment (pamandrejko) [reply docs/source/write_first_app.rst:305]: The commented out example code in invoke.js do not match the examples in the doc:
```
// Submit the specified transaction.
        // createCar transaction - requires 5 argument, ex: ('createCar', 'CAR12', 'Honda', 'Accord', 'Black', 'Tom')
        // changeCarOwner transaction - requires 2 args , ex: ('changeCarOwner', 'CAR10', 'Dave')
        //await contract.submitTransaction('createCar', 'CAR12', 'Honda', 'Accord', 'Black', 'Tom');```

For example should 
// changeCarOwner transaction - requires 2 args , ex: ('changeCarOwner', 'CAR10', 'Dave')
be changed to match the tutorial:
('changeCarOwner', 'CAR12', 'Dave');

[the remark this answers is not in the document]
Comment (pamandrejko) [reply docs/source/write_first_app.rst:305]: node query.js results (after transfer of car to Dave) should be:
`Transaction has been evaluated, result is: {"color":"Black","docType":"car","make":"Honda","model":"Accord","owner":"Dave"}`

**Review by pamandrejko — COMMENTED** [review]
I ran through all the steps on my machine using the Fabcar from FAB-17268.
It worked fine.
There are few results that are different now that we should probably update and one broken link.

File: docs/source/write_first_app.rst [hunk 2]
```diff
@@ -174,58 +148,52 @@ locally and the public key is then sent to the CA which returns an encoded
 certificate for use by the application. These three credentials are then stored
 in the wallet, allowing us to act as an administrator for the CA.
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply docs/source/write_first_app.rst]: done.

File: docs/source/write_first_app.rst [hunk 3]
```diff
@@ -174,58 +148,52 @@ locally and the public key is then sent to the CA which returns an encoded
...
-
 Let's enroll user ``admin``:
 
 .. code:: bash
 
   node enrollAdmin.js
 
-This command has stored the CA administrator's credentials in the ``wallet``
-directory.
+This command stores the CA administrator's credentials in the ``wallet`` directory.
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply docs/source/write_first_app.rst]: I will say you can find the certificate and private key in the `admin.id` file.

File: docs/source/write_first_app.rst [hunk 4]
```diff
@@ -291,14 +264,14 @@ code connects the application to a particular channel within the network,
 
   const network = await gateway.getNetwork('mychannel');
 
-Within this channel, we can access the smart contract ``fabcar`` to interact
+Within this channel, we can access the FabCar smart contract to interact
 with the ledger:
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply docs/source/write_first_app.rst:272]: mentioned that here and up top with the section with the `startFabric.sh` script

File: docs/source/write_first_app.rst [hunk 5]
```diff
@@ -291,14 +264,14 @@ code connects the application to a particular channel within the network,
...
 
 .. code:: bash
 
   const contract = network.getContract('fabcar');
 
-Within ``fabcar`` there are many different **transactions**, and our application
+Within FabCar there are many different **transactions**, and our application
 initially uses the ``queryAllCars`` transaction to access the ledger world state
 data:
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply docs/source/write_first_app.rst:280]: done.

File: docs/source/write_first_app.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikhil550) [reply docs/source/write_first_app.rst:305]: done.

**Review by mastersingh24 — APPROVED** [review]
LGTM - but will wit for @pamandrejko

**Review by pamandrejko — APPROVED** [review]
Thanks Nik

**Pull request merged**